Docker_Container/readme.md

```diff
@@ -0,0 +1,17 @@
+<strong>Just put the whole project inside this folder</strong>
```

Comment:
1. This is HTML. Convert to markdown.
1. Change the name of the file to README.md so that it will automatically format in GitHub